### ISSUE TITLE
responseXML error

### ISSUE THREAD

**Issue by weoam** [Tampermonkey/tampermonkey#1966]
## Expected Behavior
get xml document of the URL

## Actual Behavior
URL of responseXML property shows the current page's URL. in the document there's a parsererror that says error on line 21 at column 76: Specification mandates value for attribute crossorigin

## Specifications

- Chromium: 120.0.6099.199
- TM: 5.0.1
- OS: windows 10

## Script

```js
// ==UserScript==
// @name         test2
// @namespace    none
// @version      1
// @description  a
// @author       a
// @match        https://github.com/*
// @grant        GM_xmlhttpRequest
// ==/UserScript==
GM_xmlhttpRequest({
  method: "GET",
  url: "https://github.com/",
  onload: function(response) {
    console.log(response);
console.log(response.responseXML);
  }
});
```
works fine in violentmonkey

**Comment by derjanb** [comment]
This is working fine here with:

Chromium Version 120.0.6099.71
Google Chrome Version 122.0.6170.5
Chromium Version 120.0.6099.129

Independent of whether I'm logged in or not...

**Comment by derjanb** [comment]
Can you add more details? Are you using other extensions in parallel?

**Comment by weoam** [comment]
i disabled most other extensions, also tried in firefox. does a setting affect this?

**Comment by derjanb** [comment]
Can you please attach the content of `response.responseText`?
Is it the expected response? Do you see obvious syntax errors?

**Comment by weoam** [comment]
[responseText.txt](https://github.com/Tampermonkey/tampermonkey/files/13873964/responseText.txt)
seems fine

**Comment by derjanb** [comment]
Should be fixed at 5.1.6193 ([crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|xpi in review)

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions` (after you've enabled 'Developer Mode').
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
